.github/workflows/deploy-earn-protocol-app-new-production.yaml

```
@@ -0,0 +1,213 @@
+          PGPASSWORD=$SUMMER_DB_PASSWORD /usr/bin/psql -d $SUMMER_DB_NAME -U $SUMMER_DB_USER -h $SUMMER_DB_HOST -c 'SELECT 1;' > /dev/null
+          STATUS_CODE=$?
+          if ! [[ "$STATUS_CODE" = 0 ]]; then
+            echo 'VPN connection failed'
+            exit 1
+          fi
+          echo 'VPN connected!'
+
+      - name: Extract commit hash
+        id: vars
+        shell: bash
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)